src/pages/mobile/AppBrowser.vue

```diff
@@ -102,7 +103,29 @@ export default {
       },
     }),
   },
-  mounted() {
+  async mounted() {
+    const sdk = this.$store.state.sdk.then ? await this.$store.state.sdk : this.$store.state.sdk;
+
+    const shareWalletInfo = async (postFn, { interval = 5000, attemps = 5 } = {}) => {
```

Comment:
I think we should send these messages all the time: https://github.com/aeternity/aepp-sdk-js/issues/907